astilectron-bundler/main.go

```diff
@@ -7,12 +7,14 @@ import (
 	"path/filepath"
 	"runtime"
 
-	"github.com/asticode/go-astilectron-bundler"
-	"github.com/asticode/go-astilog"
-	"github.com/asticode/go-astitools/flag"
+	astibundler "github.com/asticode/go-astilectron-bundler"
+	astilog "github.com/asticode/go-astilog"
+	astiflag "github.com/asticode/go-astitools/flag"
 	"github.com/pkg/errors"
 )
 
+var ldflags = astibundler.LDFlags{}
```

Comment:
I'd rather you create a different struct in the `main` package that implements the `Set` method.

That way, the `LDFlags` struct in the `bundler` package isn't coupled with CLI flags logic.